manifests/params.pp

```diff
@@ -29,9 +29,21 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
```

Comment:
According to https://launchpad.net/ubuntu/vivid/+package/lsb-release

> "The Linux Standard Base (http://www.linuxbase.org/) is a standard
 core system that **third-party applications written for Linux can
 depend upon**"

So I think we are safe :smile: my vote is for consistency over deprecation/dependencies, in this specific case. But I agree with you in general, we should try to move away from legacy stuff as much as possible.